protected/humhub/modules/content/widgets/DeleteLink.php

```diff
@@ -31,9 +31,13 @@ class DeleteLink extends \yii\base\Widget
     public function run()
     {
         if ($this->content->content->canEdit()) {
+
+            $isAdmin = \Yii::$app->user->isAdmin() && $this->content->content->created_by !== \Yii::$app->user->id;
```

Comment:
Same here. This modal should open for everybody in Space which is allowed to delete content and is not the author.